R/stability_selection.R

```diff
@@ -90,23 +91,30 @@
 #'     uses the \code{\link[stabs]{stabsel}} function from the \code{stabs}
 #'     package, but implements the randomized lasso version.
 #'
-#' @param x the predictor matrix.
-#' @param y the response vector.
-#' @param weakness value between 0 and 1 (default = 0.8).
+#' @param x The predictor matrix.
+#' @param y The response vector.
+#' @param weakness Value between 0 and 1 (default = 0.8).
 #'     It affects how strict the method will be in selecting predictors. The
 #'     closer it is to 0, the more stringent the selection. A weakness value
 #'     of 1 is identical to performing lasso stability selection (not the
 #'     randomized version).
-#' @param cutoff value between 0 and 1 (default = 0.8) which is the cutoff
+#' @param glmnet.args Named list with additional arguments (beyond \code{x}, 
+#'     \code{y} and \code{weakness}, which are determined automatically, and 
+#'     \code{q} that can be passed via \code{...}) that 
```

Comment:
Should we also mention the `type` argument in `.glmnetRandomizedLasso()`?

Reply:
Thanks - the reason I didn't mention `type` specifically was that they way I read the code, this would neither be automatically determined nor be passed via `...` (since it's not an argument to `stabs::stabsel()`). So I think it could be provided to `glmnet.args` (and thus is not an exception). Did I misunderstand?